agent/Agent.js

```diff
@@ -213,6 +218,7 @@ class Agent extends EventEmitter {
       bridge.forget(id);
     });
     this.on('setSelection', data => bridge.send('select', data));
+    this.on('setInspectEnabled', data => bridge.send('setInspectEnabledFromBackend', data));
```

Comment:
I'd be keen to get thoughts on the naming of the `setInspectEnabledFromBackend` architecture and naming.